### DISCUSSION TITLE
Introducing Local Packages

### DISCUSSION THREAD

**Discussion by stolinski** [stolinski/drop-in#9] [Announcements]
### Packages dir

Local packages make working with monorepo style development simple. Based on Meteor's package dir, you can add local packages to ./src/packages and have them automatically added to the current monorepo.

And more, I've also added a cli tool to make a new local Svelte Package.

### make-package

Usage: `npm run make-package`

Follow the wizard to create a new package. This will add a new package starter to the local pacakges folder /src/packages. This will also add "@org-name/your-package-name": "workspace:^", to your package.json file

Basically, this feature takes a dx nicety from Meteor and finds still works within the modern JavaScript ecosystem.

### slick-kit internal packages

We're now shipping add ons and sub packages directly in the template app code. This gives you the ability to modify them with zero fuss if you want to make deeper customizations.

## Tldr

Go from zero to 100 with npm packages directly in your app code. One command and you can create a new one. Modify slick-kit's own packages for full control over your app.
